src/treeview/FavoritesProvider.ts

```diff
@@ -0,0 +1,133 @@
+import * as vscode from 'vscode';
+import { TableNode } from './TableNode';
+import { TablesListProvider } from './TablesListProvider';
+import { PanelViewProvider } from '../webview/PanelViewProvider';
+import { IConfig, TableCount } from '../view/app/model';
+import { Constants } from '../common/Constants';
+
+export class FavoritesProvider extends TablesListProvider {
+    public _onDidChangeTreeData: vscode.EventEmitter<
+        TableNode | undefined | void
+    > = new vscode.EventEmitter<TableNode | undefined | void>();
+    readonly onDidChangeTreeData: vscode.Event<TableNode | undefined | void> =
+        this._onDidChangeTreeData.event;
+    public configs: IConfig[] | undefined;
+
+    constructor(
+        fieldsProvider: PanelViewProvider,
+        indexesProvider: PanelViewProvider,
+        context: vscode.ExtensionContext
+    ) {
+        super(fieldsProvider, indexesProvider, context);
+    }
+
+    public getTreeItem(element: TableNode): vscode.TreeItem {

```

Comment:
Let's start adding jsdocs. They can be generated pressing `ctrl+space` on the line above the method/function: 
![image](https://github.com/BalticAmadeus/ProBro/assets/44312404/70664ca0-6772-4365-96f6-4ade67131c88)